### PR TITLE
Added few missing mappings

### PR DESCRIPTION
**Pull request by mmozeiko**
Adds few missing tvdb mappings, movie mappings.

And fixes Street Fighter IV: Aratanaru Kizuna mapping to correct tvdbid episode number.

### CODE REVIEW

File: anime-list-master.xml [hunk 1]
```diff
@@ -41743,7 +41743,7 @@
   <anime anidbid="15144" tvdbid="388737" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Jaku-chara Tomozaki-kun</name>
   </anime>
-  <anime anidbid="15145" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="15145" tvdbid="371000" defaulttvdbseason="1" episodeoffset="" tmdbid="94498" imdbid="tt11714178">
```
Comment (reconman) [inline anime-list-master.xml:41746]: This tmdbid belongs to https://www.themoviedb.org/movie/94498-i-love-dollars

File: anime-list-master.xml [hunk 2]
```diff
@@ -38449,7 +38449,7 @@
   <anime anidbid="13991" tvdbid="337936" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="">
     <name>Karakai Jouzu no Takagi-san: Water Slider</name>
   </anime>
-  <anime anidbid="13992" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="13992" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="514710" imdbid="tt8223844">
```
Comment (reconman) [inline anime-list-master.xml:38452]: This one does not have tvdbid="movie" set.

**Review by reconman — CHANGES_REQUESTED** [review]

File: anime-list-master.xml (hunk 1 above)
Comment (mmozeiko) [reply anime-list-master.xml]: Ooops, no idea why I added tmdb/imdb there. No need for those. Just tvdbid for tv show is enough. Fixed.

File: anime-list-master.xml (hunk 2 above)
Comment (mmozeiko) [reply anime-list-master.xml]: Missed this one. Fixed it.

**Comment by reconman** [comment]
I would appreciate if you don't force push after someone reviewed your PR, since that way I can't tell what your latest changes are.

I can combine the commits into 1 with the `Squash and Merge` button, anyway.

**Pull request merged**